### PR TITLE
Use slugs in <Link/> components

### PR DESCRIPTION
**Pull request by peitalin**
I've cleaned up the commit history a bit with a new branch. This is the same PR as before.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -59,3 +62,30 @@ export const abbreviatePrice = (number: string) => {
...
+
+export function slugToAddress(slugOrAddress: string, chainId: ChainId) {
+  if (utils.isAddress(slugOrAddress)) {
+    return slugOrAddress;
+  }
+  if (!collections?.[chainId]) {
+    return slugOrAddress;
+  }
+  const tokenAddress = collections?.[chainId]?.find(
+    (t) => slugOrAddress === t.slug
```
Comment (jcheese1) [inline src/utils.ts:74]: can`t.slug` be `getCollectionSlugFromName(t.name)` and remove the slug field all together?

Comment (peitalin) [reply src/utils.ts:74]: Good idea, let me make those changes and remove the slug field altogether

File: src/components/Header.tsx [hunk 2]
```diff
@@ -190,9 +191,13 @@ const Header = () => {
                         )
                         .map((collection) => {
                           const active = collection.address === address;
+                          const slugOrAddress =
```
Comment (jcheese1) [inline src/components/Header.tsx:194]: oh, and we also use `<SearchAutocomplete />` in `index.tsx`, mind changing that part too with this logic?

Comment (peitalin) [reply src/components/Header.tsx:194]: yep, sure thing.

**Comment by jcheese1** [comment]
@peitalin thanks for all the revision! looks good to me, @wyze will do the final check tmrw

**Comment by peitalin** [comment]
Nice! ty for reviewing

**Review by wyze — APPROVED** [review]
Works great, thank you!

**Pull request merged**